extensions/wikia/SoundCloudTag/SoundCloudTagController.class.php

```diff
@@ -0,0 +1,44 @@
+<?php
+class SoundCloudTagController extends WikiaParserTagController {
+	const PARSER_TAG_NAME = 'soundcloud';
+	const API_ENDPOINT = 'https://w.soundcloud.com/player/?';
+
+
+	public static function onParserFirstCallInit( Parser $parser ) {
+		$parser->setHook( self::PARSER_TAG_NAME, [ new static(), 'renderTag' ] );
+		return true;
+	}
+
+	public function renderTag( $input, array $args, Parser $parser, PPFrame $frame ) {
+		return Html::element(
+			'iframe',
+			$this->prepareAttributes( $args ),
+			wfMessage( 'soundcloud-tag-could-not-render' )->text()
+		);
+	}
+
+	private function prepareAttributes( array $args ) {
+		$allowedArgs = ['width', 'height', 'scrolling', 'frameborder', 'src'];
```

Comment:
Please use the same list as in https://github.com/Wikia/app/pull/8198/files#diff-f9128714a37a935b74cae60b94783da9R24

Also user doesn't have to click "Embed" and copy the iframe, `url` parameter (to track/playlist/etc) is enough.